modules/fedimint-ln-server/src/lib.rs

```diff
@@ -306,16 +329,24 @@ impl ServerModule for Lightning {
         consensus_items: Vec<(PeerId, LightningConsensusItem)>,
         _consensus_peers: &BTreeSet<PeerId>,
     ) -> Vec<PeerId> {
-        for (peer, decryption_share) in consensus_items.into_iter() {
+        let mut peer_heights = dbtx.get_value(&BlockHeightKey).await.unwrap_or_default();
```

Comment:
What are the implications of peers having a different BlockHeight? I would imagine this would be a short lived situation?

Reply:
We use the median of block height between peers which works if enough honest peers have contributed.